app/res/values/strings.xml

```diff
@@ -456,4 +456,5 @@
     <string name="fcm_default_notification_channel">notification-channel-push-notifications</string>
     <string name="app_with_id_not_found">Required CommCare App is not installed on device</string>
     <string name="audio_recording_notification">Audio Recording Notification</string>
+    <string name="logs_server_url_protocol">https://</string>
```

Comment:
I think this is going to be really confusing if this is the default value. 

From my perspective and the discussion below, we need to frame this as an "override" setting, which redirects logs to a separate path. Having any value in here (like a stub URL) feels very confusing, as it implies that the value was in use before.

I think the "baseline" value of this setting should be an empty string (``) so it's clear that its clear when it's unused, but more critically so that it's clearer on how to reset back to empty.

Reply:
I was trying to find a way to inform the user that the protocol needs to be `https` as it will fail otherwise. But happy to explore another option